examples/nlp/language_modeling/megatron_ckpt_to_nemo.py

```diff
@@ -73,15 +77,26 @@ def convert(rank, world_size, args):
 
     app_state = AppState()
     app_state.data_parallel_rank = 0
-    trainer = Trainer(gpus=args.tensor_model_parallel_size)
-    # TODO: reach out to PTL For an API-safe local rank override
-    trainer.accelerator.training_type_plugin._local_rank = rank
+    trainer = Trainer(gpus=args.tensor_model_parallel_size * args.pipeline_model_parallel_size)
```

Comment:
what if `args.tensor_model_parallel_size * args.pipeline_model_parallel_size` > number of gpus per node?
We need to set the `num_of_nodes`?